cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,50 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import SettingsPageObject from '../support/pages/settings.pageObject';
+
+const settingsPage = new SettingsPageObject();
+
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.visit('');
+    cy.register();
+    cy.login();
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
-
+    settingsPage.editUsername(user.username);
+    settingsPage.submitEdit();
+    settingsPage.verifyProfilePage(user.username);
```

Comment:
The `verifyProfilePage` method is called with `user.username`, but the default parameter in `verifyProfilePage` is 'riot'. Ensure that the `verifyProfilePage` method in `settings.pageObject.js` is correctly using the passed `username` parameter.